src/language-service/src/schemas/triggers.ts

```diff
@@ -371,3 +372,8 @@ interface ZoneTrigger {
    */
   event: "enter" | "leave";
 }
+
+interface TagTrigger {
+  platform: "tag";
+  tag_id: string;
+}
```

Comment:
Missing documentation for each option.

Reply:
Yes, it's why my comment within the PR description.

Finally, I found a description here: https://www.home-assistant.io/integrations/tag/